opennms-config/src/test/resources/poller-configuration.xml

```diff
@@ -0,0 +1,512 @@
+<?xml version="1.0"?>
+<poller-configuration threads="30"
+                      serviceUnresponsiveEnabled="false"
+                      nextOutageId="SELECT nextval('outageNxtId')">
+  <node-outage status="on" pollAllIfNoCriticalServiceDefined="true">
+    <critical-service name="ICMP" />
+  </node-outage>
+  <package name="example1">
+    <filter>IPADDR != '0.0.0.0'</filter>
+    <specific>0.0.0.0</specific>
+    <!--<include-range begin="1.1.1.1" end="254.254.254.254" />-->
+    <include-url>file:${INCLUDE_URL_FILE}</include-url>
+    <rrd step="300">
+      <rra>RRA:AVERAGE:0.5:1:2016</rra>
+      <rra>RRA:AVERAGE:0.5:12:1488</rra>
+      <rra>RRA:AVERAGE:0.5:288:366</rra>
+      <rra>RRA:MAX:0.5:288:366</rra>
+      <rra>RRA:MIN:0.5:288:366</rra>
+    </rrd>
+    <service name="ICMP" interval="300000" user-defined="false" status="on">
```

Comment:
Can we trim this file down to only keep what's necessary? (Seems like it's only used to testing the `include-url`, so we don't need all of the services / monitors defined.)